webapp/portlet/src/main/webapp/vue-apps/space-top-bannerlogo/components/ExoSpaceLogoBanner.vue

```diff
@@ -10,6 +10,7 @@
       :nudge-width="200"
       max-width="350"
       min-width="300"
+      content-class="no-box-shadow full-height pa-1 mt-6 "
```

Comment:
```suggestion
      content-class="no-box-shadow full-height pa-1"
```
mt-6 is not necessary

Reply:
> mt-6 is not necessary

It's necessary to add mt-6

Reply:
Can you explain why ? When I remove it on my local test, I see it is better without

Reply:
> Can you explain why ? When I remove it on my local test, I see it is better without

The popover space isn't positioned below the space name due to full-height, so I added the mt-6  to fix the popover's positioning.